x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/dashboard_sections/benchmarks_section.tsx

```diff
@@ -82,8 +83,12 @@ export const BenchmarksSection = ({
                       </EuiText>
                     </EuiFlexItem>
                     <EuiFlexItem grow={false}>
-                      {/* TODO: change default k8s logo to use a getBenchmarkLogo function */}
-                      <EuiIcon type="logoKubernetes" size="xxl" />
+                      {cluster.meta.benchmarkId === 'cis_eks' && (
+                        <EuiIcon type={cisEksIcon} size="xxl" />
+                      )}
+                      {cluster.meta.benchmarkId === 'cis_k8s' && (
+                        <EuiIcon type={cisK8sVanillaIcon} size="xxl" />
+                      )}
```

Comment:
```suggestion
<EuiIcon type={getBenchmarkIconType(cluser.meta.benchmarkId)} size="xxl" />
```
`getBenchmarkIconType` can be exported and reused in places where we use those icons